### PR TITLE
allows downloading of badges in zipped, pdf and svg versions

### LINKED ISSUE #9: Allow downloading of badges in multiple formats
Allow downloading of badges in multiple formats:

- [ ] ZIP file containing all SVGs and PDFs
- [ ] SVG
- [ ] PDF of individual set

### PR DESCRIPTION
**Pull request by yashhere**
Fixes #9 

#### Short description of what this resolves:
This PR creates a zip file containing all-badges.zip, and all the pdf and SVG in the folder of badges. Other files generated are a pdf file of all the badges and a zip of all the SVG generated

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -6,10 +6,16 @@
 			{% with m = get_flashed_messages(with_categories=true) %}
 				{% if m %}
 					{% for cat,msg in m %}
-						<div class="flash-{{cat}}">{{msg}}</div>
+						<div class="flash-{{cat}}">Your badges has been successfully generated.</div>
```
Comment (jajodiaraghav) [inline app/templates/index.html]: This will show "Your badges has been successfully generated." for the error msg as well

File: app/main.py [hunk 2]
```diff
@@ -55,15 +58,17 @@ def upload():
...
-    	return redirect(url_for('index'))
+        flash('Only CSV file is accepted!', 'error')
+        return redirect(url_for('index'))
+
 
 @app.errorhandler(404)
 def Not_Found(e):
-	return render_template('404.html'), 404
+    return render_template('404.html'), 404
+
```
Comment (jajodiaraghav) [inline app/main.py]: Please refrain from adding newlines unnecessarily!

**Comment by jajodiaraghav** [comment]
Great work @yashhere 
Left a few comments. Will merge once you address the issues :)

Thanks a lot :)

**Comment by niccokunzmann** [comment]
@jajodiaraghav Thanks for merging and taking care. Whenever you think, someone is here to help, you can also create an issue to ask them to become maintainers. For me, maintainers do not need to know all the code or be good at programming but willing to keep the project going :)

**Comment by jajodiaraghav** [comment]
@niccokunzmann Okay. Will keep this in mind.

**Pull request merged**